source/slides/05_dns_dhcp.rst

```diff
@@ -65,6 +101,73 @@ They look like::
 
 * They inform where to direct DNS queries for a domain
 * Point to other domains (which have A records)
+* Glue records are sometimes needed and provided to registrar
```

Comment:
This is a good time to point out the problem glue records solve, and then ask how the students would solve it, then discuss what glue records are, etc.